Makefile

```diff
@@ -155,7 +157,7 @@ copy_node_js:
 ifeq ($(IN_DOCKER),)
 	$(warning Command is designed to be run in the container)
 endif
-	for dest in $(NODE_LIBS_JS) ; do cp $(NODE_MODULES)$$dest $(STATIC_JS) ; done
+	for dest in $(NODE_LIBS_JS) ; do cp -r $(NODE_MODULES)$$dest $(STATIC_JS) ; done
```

Comment:
https://github.com/mozilla/addons-server/pull/5581/commits/b38912c42a998c1af478c0fd54d4fc914f0c54fb
(I rebased it into a single commit due to merge conflict issues)